File: QuevedoVirtualClassrooms-Server/src/main/java/com/quevedo/virtualclassroomsserver/common/models/dto/resource/ResourceCommentDTO.java
package com.quevedo.virtualclassroomsserver.common.models.dto.resource;

import lombok.Builder;
import lombok.Data;

import java.time.LocalDateTime;

@Data
@Builder
public class ResourceCommentDTO {
    private String usernameOwner;
    private String text;
    private LocalDateTime timeStamp;
    private boolean isAnswer;
    private String answersTo;
    private String resourceUUID;
}

File: QuevedoVirtualClassrooms-Android/app/src/main/java/org/quevedo/quevedovirtualclassrooms/data/models/resource/ResourceCommentDTO.kt
package org.quevedo.quevedovirtualclassrooms.data.models.resource

import java.time.LocalDateTime

data class ResourceCommentDTO(
    val idComment: Int,
    val usernameOwner: String,
    val text: String,
    val timeStamp: LocalDateTime,
    val answersTo: String,
)